src/Findologic/Resource/ServiceConfigResource.php

```diff
@@ -64,15 +64,23 @@ public function isStaging(?string $shopkey): bool
         return $this->get($shopkey, 'isStagingShop');
     }
 
+    /**
+     * @throws InvalidArgumentException
+     * @throws ClientException
+     */
+    public function getSmartSuggestBlocks(?string $shopkey): array
+    {
+        return $this->get($shopkey, 'blocks');
+    }
+
     /**
      * @throws InvalidArgumentException
      */
     private function getFromCache(): ?ServiceConfig
     {
-        /** @var ServiceConfig $config */
         $config = $this->cache->getItem(self::CACHE_KEY)->get();
         if ($config !== null) {
-            return unserialize($config, [ServiceConfig::class]);
```

Comment:
Why was this changed and why was the docblock removed?

Reply:
the docblock was removed as the `$config` variable should be a serialized string from the cache instead of the object, so it was incorrect. And the class name change I will revert it.

Reply:
The class name change has not been reverted.